dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -3547,6 +3555,14 @@ public void run() {
         }
     }
 
+    private void updateContentletTemplate(Contentlet c, String newTemplate, Field fieldVar) throws DotDataException {
+        contentFactory.updateContentletTemplate(c.getInode(), newTemplate,
+                fieldVar.getFieldContentlet());
+        CacheLocator.getContentletCache().remove(c);
+        CacheLocator.getHTMLPageCache().remove(c.getInode());
```

Comment:
I think the getHTMLPageCache takes an identifier?  Or is this correct?

Reply:
Take a look at https://github.com/dotCMS/enterprise-2.x/blob/master/src/main/java/com/dotcms/enterprise/publishing/remote/handler/HandlerUtil.java#L114